Makefile

```diff
@@ -209,15 +209,26 @@ test: all
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
 	$(MAKE) doc
+	$(MAKE) lint
 	$(MAKE) cctest
 	$(PYTHON) tools/test.py --mode=release -J \
 		$(CI_ASYNC_HOOKS) \
 		$(CI_JS_SUITES) \
 		$(CI_NATIVE_SUITES) \
 		known_issues
-	$(MAKE) lint
 endif
 
+# For a quick test, does not run linter or build doc
+t: all
```

Comment:
`t` is a terrible name for a make target (that what it does has to be described in a comment is a good indicator that it is badly named). We're already at 40+ phony targets -- It should be in our interest that the target names be reasonably self descriptive.

Reply:
But this isn't something most people will want to use. It looks like a shortcut, which is what it is.

Reply:
I would prefer `test-only`, personally.